SystemInfo/Value.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+/**
+ * File containing the Value class.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\EzSupportToolsBundle\SystemInfo;
+
+use eZ\Publish\API\Repository\Values\ValueObject;
+
+/**
+ * System information Value.
+ */
+abstract class Value extends ValueObject
```

Comment:
@dpobel I asked @bdunogier that, he wanted a common one for typehinting. Though I suppose we *could* use ValueObject directly..?